config/demo.exs

```diff
@@ -8,3 +8,570 @@ config :wanda, WandaWeb.Endpoint,
+        target5 => 10,
+        target6 => 10
+      }
+    },
+    "00081D" => %{
+      "max_messages" => %{
+        target1 => 20,
+        target2 => 20,
+        target3 => 20,
+        target4 => 20,
+        target5 => 20,
+        target6 => 20
+      }
+    },
+    "32CFC6" => %{
+      "totem_interfaces" => %{
+        target1 => [
+          %{
+            "bindnetaddr" => "10.80.1.12",
+            "mcastaddr" => "239.192.100.25",
+            "mcastport" => 5405,
+            "ttl" => 1
+          }
+        ],
+        target2 => [
+          %{
+            "bindnetaddr" => "10.80.1.12",
+            "mcastaddr" => "239.192.100.25",
+            "mcastport" => 5405,
+            "ttl" => 1
+          }
+        ],
+        target3 => [
+          %{
+            "bindnetaddr" => "10.80.1.12",
+            "mcastaddr" => "239.192.100.25",
+            "mcastport" => 5405,
+            "ttl" => 1
+          }
+        ],
+        target4 => [
+          %{
+            "bindnetaddr" => "10.80.1.12",
+            "mcastaddr" => "239.192.100.25",
+            "mcastport" => 5405,
+            "ttl" => 1
+          }
+        ],
+        target5 => [
+          %{
+            "bindnetaddr" => "10.80.1.12",
+            "mcastaddr" => "239.192.100.25",
+            "mcastport" => 5405,
+            "ttl" => 1
+          }
+        ],
+        target6 => [
+          %{
+            "bindnetaddr" => "10.80.1.12",
+            "mcastaddr" => "239.192.100.25",
+            "mcastport" => 5405,
+            "ttl" => 1
+          }
+        ]
+      }
+    },
+    "15F7A8" => %{
+      "token_retransmits" => %{
+        target1 => 10,
+        target2 => 10,
+        target3 => 10,
+        target4 => 10,
+        target5 => 10,
+        target6 => 10
+      }
+    },
+    "FB0E0D" => %{
+      "consensus_timeout" => %{
+        target1 => 36_000,
+        target2 => 36_000,
+        target3 => 36_000,
+        target4 => 36_000,
+        target5 => 36_000,
+        target6 => 36_000
+      }
+    },
+    "D78671" => %{
+      "runtime_two_node" => %{
+        target1 => 1,
+        target2 => 1,
+        target3 => 1,
+        target4 => 1,
+        target5 => 1,
+        target6 => 1
+      }
+    },
+    "373DB8" => %{
+      "crm_config_properties" => %{
+        target1 => %{"acme" => 42},
+        target2 => %{"acme" => 42},
+        target3 => %{"acme" => 42},
+        target4 => %{"acme" => 42},
+        target5 => %{"acme" => 42},
+        target6 => %{"acme" => 42}
+      },
+      "resources_primitives" => %{
+        target1 => %{"foo" => [%{"bar" => "baz"}]},
+        target2 => %{"foo" => [%{"bar" => "baz"}]},
+        target3 => %{"foo" => [%{"bar" => "baz"}]},
+        target4 => %{"foo" => [%{"bar" => "baz"}]},
+        target5 => %{"foo" => [%{"bar" => "baz"}]},
+        target6 => %{"foo" => [%{"bar" => "baz"}]}
+      }
+    },
+    "816815" => %{
+      target1 => %{"sbd_service_state" => "active"},
```

Comment:
let's double check this. the structure is not correct.

Reply:
Thanks i have a look :D